books/free-programming-books-subjects.md

```diff
@@ -751,6 +786,21 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Your API Is Bad](https://leanpub.com/yourapiisbad/read) - Paddy Foran
 
 
+### Software Architecture - Design Patterns
```

Comment:
"Design Patterns" has a specific meaning that excludes, for example, "domain-driven design".